test/integration/compliance_history_test.go

```diff
@@ -545,11 +533,16 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
 			// It can take a while for the Gatekeeper audit pod to produce audit results.
 		}, defaultTimeoutSeconds*6, 1).Should(Succeed())
 
-		By("Updating the ConfigMap to be valid")
-		configmap.Labels = map[string]string{"my-gk-test": "a-value"}
-		_, err = clientHub.CoreV1().ConfigMaps(gkTargetNS).Update(ctx, configmap, metav1.UpdateOptions{})
+		By("Updating the ConfigMap to be valid with a policy")
+		_, err = common.OcHub(
+			"apply",
+			"-f",
+			"../resources/compliance_history/policy-gk-prereq2.yaml",
```

Comment:
We need a clean up for this

Reply:
@yiraeChristineKim the previous `DeferCleanup` covers this since it's the same policy name being used.

Reply:
Great!

Reply:
I added a comment to make this obvious